app/lib/widgets/wallets/add_edit_contact.dart

```diff
@@ -232,9 +238,49 @@ class _AddEditContactState extends State<AddEditContact> {
                   ),
                   controller: _addressController,
                 ),
-                const SizedBox(
-                  height: 30,
-                ),
+                const SizedBox(height: 20),
+                if (widget.operation == ContactOperation.Add)
+                  Row(
+                    children: [
+                      Text(
+                        'Chain Type:',
+                        style: Theme.of(context).textTheme.bodyMedium!.copyWith(
+                            color: Theme.of(context).colorScheme.onSurface,
+                            decorationColor:
+                                Theme.of(context).colorScheme.onSurface),
+                      ),
+                      const SizedBox(width: 10),
+                      DropdownButton<String>(

```

Comment:
please use the `chainType` directly in the dropdown
```suggestion
                      DropdownButton<ChainType>(

```